mods/ra/maps/allies-06b/allies06b.lua

```diff
@@ -220,15 +220,15 @@ WorldLoaded = function()
 				end
 			end
 		end)
+	end
 
-		if Map.LobbyOption("difficulty") == "easy" then
-			Trigger.OnKilled(DefBrl1, function(a, b)
-				DefenseFlame1.Kill()
-			end)
-			Trigger.OnKilled(DefBrl2, function(a, b)
-				DefenseFlame2.Kill()
-			end)
-		end
+	if Map.LobbyOption("difficulty") ~= "hard" then
+		Trigger.OnKilled(DefBrl1, function(a, b)
+			DefenseFlame1.Kill()
```

Comment:
We should probably add an `IsDead` check here to prevent a crash when you destroy the flamers before touching the barrels.